src/aol-hb.js

```diff
@@ -27,4 +32,27 @@ globalContext.init = (bidRequestConfig, placementsConfigs) => {
   };
 };
 
+globalContext.que.push = (cmd) => {
+  if (typeof cmd === 'function') {
+    try {
+      cmd.call();
+    } catch (e) {
+      console.warn('Error processing command :' + e.message);
+    }
+  } else {
+    console.warn('Commands written into aolhb.que.push must wrapped in a function');
+  }
+};
+
+for (let i = 0; i < globalContext.que.length; i++) {
+  if (typeof globalContext.que[i].called === 'undefined') {
+    try {
+      globalContext.que[i].call();
+      globalContext.que[i].called = true;
+    } catch (e) {
+      console.warn('Error processing command :', 'aol-hb.js', e);
```

Comment:
I would suggest removing space before the colon. Having `aol-hb.js` in the console message is a nice idea. Do we want to apply that to all console logs?